usage/rest-server/src/test/java/brooklyn/rest/BrooklynRestApiLauncher.java

```diff
@@ -235,6 +238,7 @@ private ContextHandler servletContextHandler(ManagementContext managementContext
         ResourceConfig config = new DefaultResourceConfig();
         for (Object r: BrooklynRestApi.getAllResources())
             config.getSingletons().add(r);
+        config.getSingletons().add(new ShutdownRequestListenerProvider(new TestShutdownRequestListener()));
```

Comment:
Update code to use the field. The field itself is not used elsewhere, but added it anyway for consistency.